src/mmrelay/db_utils.py

```
@@ -565,77 +765,183 @@ def _initialize(cursor: sqlite3.Cursor) -> None:
+        elif stale_table_exists and temp_exists:
+            cursor.execute(_PRAGMA_MESSAGE_MAP_TEMP_INFO_SQL)
+            temp_columns = {column[1] for column in cursor.fetchall()}
+            cursor.execute(_PRAGMA_MESSAGE_MAP_STALE_TEMP_INFO_SQL)
+            stale_columns = {column[1] for column in cursor.fetchall()}
+            if _col_mesh in temp_columns:
+                stale_mesh_expr = _col_mesh if _col_mesh in stale_columns else "NULL"
+                cursor.execute(
+                    f"INSERT OR IGNORE INTO {_temp_table} "  # nosec B608
+                    f"({_col_id}, {_col_evt}, {_col_room}, {_col_text}, {_col_mesh}) "
+                    f"SELECT {_col_id}, {_col_evt}, {_col_room}, {_col_text}, {stale_mesh_expr} "
+                    f"FROM {_MESSAGE_MAP_STALE_TEMP_TABLE}"
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)